rust-runtime/aws-smithy-client/src/hyper_impls.rs

```diff
@@ -122,6 +122,24 @@ fn find_source<'a, E: Error + 'static>(err: &'a (dyn Error + 'static)) -> Option
 
 #[derive(Default, Debug)]
 /// Builder for [`HyperAdapter`]
+///
```

Comment:
Should this doc comment go on HyperAdapter itself? At least when I was digging into the docs, that's where I landed first, and then made my way into the builder doc once I knew I wanted to use it and needed to construct it.